python-frontend/src/test/java/org/sonar/python/tree/PythonTreeMakerTest.java

```diff
@@ -2812,7 +2812,11 @@ void except_group_multiple() {
   @Test
   void typeAliasStatement() {
     setRootRule(PythonGrammar.TYPE_ALIAS_STMT);
-    var typeAlias = parse("type A[B] = str", treeMaker::typeAliasStatement);
+
+    var astNode = p.parse("type A[B] = str");
```

Comment:
Because we can't use the `RuleTest#parse` method anymore I guess we lose the visit and the test that makes sure all children are computed in order.
I realize for simple statements we have the `verify_expected_statement` that will do that by putting all simple statements in a file and check the file. Maybe we can add a type statement there?